.github/workflows/ci.yml

```diff
@@ -221,23 +229,27 @@ jobs:
               ../build/examples/trixi_controller_data_c . libelixir_p4est2d_dgsem_euler_sedov.jl
           JULIA_DEPOT_PATH=~/.julia \
               ../build/examples/trixi_controller_data_f . libelixir_p4est2d_dgsem_euler_sedov.jl
-      - name: Process coverage data
+      - name: Process C coverage data
         if: ${{ matrix.test_type == 'coverage' }}
         run: |
           cd libtrixi-julia
           lcov --directory ../build  --capture --output-file lcov.info
+      - name: Process julia coverage data
```

Comment:
```suggestion
      - name: Process Julia coverage data
        if: ${{ matrix.test_type == 'coverage' }}
```

Reply:
Let's commit this once the current checks have passed and then merge immediately.